CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
-Contributing to Bitcoin Core
-============================
+# Contributing to Bitcoin Classic
 
-The Bitcoin Core project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, testing and patches. This document explains the practical process and guidelines for contributing.
+Bitcoin Classic is hard forking bitcoin to a 2 MB blocksize limit. Please come and join us.
 
-Firstly in terms of structure, there is no particular concept of “Core developers” in the sense of privileged people. Open source often naturally revolves around meritocracy where longer term contributors gain more trust from the developer community. However, some hierarchy is necessary for practical purposes. As such there are repository “maintainers” who are responsible for merging pull requests as well as a “lead maintainer” who is responsible for the release cycle, overall merging, moderation and appointment of maintainers.
+ - [Contribution Guidelines](#contribution-guidelines)
+ - [Communication](#communication)
+ - [Merge Window](#merge-window)
+ - [Consider.it](#considerit)
+ - [Code of Conduct](#codeofconduct)
 
+## <a name="contribution-guidelines"></a> Contribution Guidelines
 
-Contributor Workflow
---------------------
+**Any change to Bitcoin must be respected by 3 groups:**
 
-The codebase is maintained using the “contributor workflow” where everyone without exception contributes patch proposals using “pull requests”. This facilitates social contribution, easy testing and peer review.
+- Developers: which implement changes
```

Comment:
I think we need another category here: 

    Maintainers: who merge those changes

Later, we describe that the role of the maintainers is to evaluate and interpret the wills of the users and the miners. Maintainers are basically equivalent to the judicial branch of government. Devs write the pull requests, and users and miners vote on policies and pull requests, but sometimes what users vote on is different from what the actual code is, and other times the pull request is in violation with a principle or proposal which has also been voted on.